.github/workflows/build-deploy-ec2.yml

```diff
@@ -52,10 +52,10 @@ jobs:
         working-directory: ./app
     steps:
       - uses: actions/checkout@v3
-      - name: Use Node.js 16.x
+      - name: Use Node.js LTS
         uses: actions/setup-node@v3
         with:
-          node-version: "16.x"
+          node-version: 'lts/*'
```

Comment:
Is there any downside to writing `node-version: 'lts/*'` vs. specifying the LTS version e.g. `node-version: '20.x'`?

If the LTS version switches to 21, could our app break? Do we benefit from having more control (i.e. specificity)? You did a bunch of research so deferring to you!